Core.DomainModel/Organization/OrganizationUnit.cs

```diff
@@ -239,5 +259,14 @@ public void ResetParent()
         {
             Parent = null;
         }
+
+        public UnitAccessRights GetAccessRights(bool isUserAllowedToRead, bool isUserAllowedToModify, bool isUserAllowedToDelete)
```

Comment:
it's not a good decision to push the "read" info down here.. if the user is not even allowed to read, then this method should not go this par...

the same goes for modification.
If user has no modification access, then who cares about the business rules...

We need to seperate RBAC from business state so "Access rights" is not part of the domain